classification/BOWClassifier.py

```diff
@@ -0,0 +1,226 @@
+import json
+import numpy as np
+import sys
+
+sys.path.append("..")
+from utils.dataset_db import dynamo_db
+
+
+def main():
```

Comment:
samples can be written into jupyter notebook for  a e s t h e t i c s  ✨ and good examples